rust/experimental/query_engine/engine-recordset-otlp-bridge/tests/otlp_kql_recordset.rs

```diff
@@ -0,0 +1,81 @@
+use data_engine_expressions::*;
+use data_engine_recordset_otlp_bridge::*;
+use data_engine_recordset2::*;
+
+use crate::common::*;
+
+pub mod common;
+
+#[test]
+fn test_project_keep() {
```

Comment:
@drewrelmas Just FYI I have a couple integration tests here which do _everything_. Construct OTLP messages. Parse KQL. Run the engine. Verify results. If you ever want to play with things end-to-end this is the best spot we have currently. Also a good spot to capture our core scenarios 😄